client/app/composer.json

```diff
@@ -72,20 +71,9 @@
         "twig/intl-extra": "^3.3",
         "web-token/jwt-core": "^2.2"
     },
-    "scripts": {
```

Comment:
This was left over on the client side. It's not needed any more now we're handling the parameters.yml at build time